universal/router/matches/WithSubRoutes.js

```diff
@@ -0,0 +1,13 @@
+/* @flow */
+import React from 'react'
+import { Match } from 'react-router'
+
+import type { RouteProps } from '../types'
+
+const WithSubRoutes = (route: RouteProps) => (
+  <Match {...route} render={(props) => (
+    <route.component {...props} store={route.store} routes={route.routes} />
+  )} />
+)
+
+export default WithSubRoutes
```

Comment:
This is directly from their recommended implementation when using a route config. The only thing i added was passing through the store for reducer injection. See https://react-router.now.sh/route-config